### PR TITLE
docs(gen): docs content review - week 33/2023

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: bare-metal/dedibox/quickstart.mdx [hunk 1]
```diff
@@ -73,7 +73,7 @@ However, you may also connect to your server via a graphical interface such as W
 You can also use FTP, SFTP or SCP to transfer data to and from your server.
 
 <Message type="requirement">
-  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
```
Comment (RoRoJ) [inline bare-metal/dedibox/quickstart.mdx:76]:
```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```

File: dedibox/kvm-over-ip/how-to/dell-idrac8.mdx [hunk 2]
```diff
@@ -48,14 +48,14 @@ The connection URL and your credentials display. Click on the link to access the
     <Lightbox src="scaleway_idrac8_connect_vm.webp" alt="" />
 5. Click **Virtual Media** > **Map CD/DVD ...**. A pop-up displays.
     <Lightbox src="scaleway_idrac8_map_cd.webp" alt="" />
-6. Seelct the installation ISO file on your computer by clicking **Browse**. Then click **Map Device** and reboot the server. 
-    <Lightbox src="scaleway_idrac8_iso.webp" alt="" /> 
-7. Press **F12** during the boot process to enter the boot manager. Then select the virtual CD-ROM drive. 
+6. Seelct the installation ISO file on your computer by clicking **Browse**. Then click **Map Device** and reboot the server.
```
Comment (RoRoJ) [inline dedibox/kvm-over-ip/how-to/dell-idrac8.mdx:51]:
```suggestion
6. Select the installation ISO file on your computer by clicking **Browse**. Then click **Map Device** and reboot the server.
```

File: tutorials/encrypt-s3-data-rclone/index.mdx [hunk 3]
```diff
@@ -230,15 +230,15 @@ Choose a number from below, or type in your own value
     \ "standard"
   2 / Very simple filename obfuscation.
     \ "obfuscate"
-  3 / Don't encrypt the file names.  Adds a ".bin" extension only.
+  3 / Do not encrypt the file names.  Adds a ".bin" extension only.
```
Comment (RoRoJ) [inline tutorials/encrypt-s3-data-rclone/index.mdx:233]: Should this be "Add" ?

Comment (bene2k1) [inline tutorials/encrypt-s3-data-rclone/index.mdx:233]:
```suggestion
  3 / Don't encrypt the file names.  Adds a ".bin" extension only.
```

Comment (bene2k1) [reply tutorials/encrypt-s3-data-rclone/index.mdx:233]: This a copy of the output of the tool. I re-edit the message back to what it was

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**